backend/curriculum_tracking/api_views.py

```diff
@@ -25,6 +24,8 @@
 from django.db.models import Q
 
 from rest_framework import filters
+from curriculum_tracking.management.helpers import get_team_cards
```

Comment:
Not used anywhere.
```suggestion
```

Reply:
Hello

I made the requested changes.  As for the tests that fail, I don't know, we worked on Andy's ticket last time.  I see that the tests that fail have to do with WorkshopAttendanceViewsetTests, I did not work on those things so I really don't know whats up with them but I will see if maybe I can figure out something:

```
Traceback (most recent call last):
  File "/home/eckard/PycharmProjects/Tilde/backend/test_mixins.py", line 48, in test_get_list
    instance = self.verbose_instance_factory()
  File "/home/eckard/PycharmProjects/Tilde/backend/curriculum_tracking/tests/test_api_views.py", line 349, in verbose_instance_factory
    workshop = factories.ContentItemFactory(content_type=ContentItem.WORKSHOP)
AttributeError: module 'core.tests.factories' has no attribute 'ContentItemFactory'
```

Reply:
The issue with all those failing tests is that the person who wrote them are trying to import from the wrong factories.py file.  I've now gone and imported everything and now all those tests pass.  However, there are some other tests like this one "def _test_set_due_time_permissions(self, card, get_progress):" that still have errors, I don't want to go and fool around with them because they are not mine.